### PR TITLE
chore: add network policy

### PR DESCRIPTION
**Pull request by SdgJlbl**
## Description

Fixes FL-1457

## Companion PR

- https://github.com/Substra/orchestrator/pull/409

## How has this been tested?

CI below

## Checklist

- [ ] [changelog](../CHANGELOG.md) was updated with notable changes
- [ ] documentation was updated

### CODE REVIEW

**Comment by Owlfred** [comment]
End to end tests: :hourglass_flowing_sand: **STARTING**

**Comment by Owlfred** [comment]
End to end tests: :hourglass_flowing_sand: **STARTING**

**Comment by Owlfred** [comment]
End to end tests: :hourglass_flowing_sand: **STARTING**

**Comment by Owlfred** [comment]
End to end tests: :hourglass_flowing_sand: **STARTING**

**Comment by guilhem-barthes** [comment]
/e2e --refs orchestrator=chore/network-policy --tests sdk,substrafl,frontend,documentation

**Comment by Owlfred** [comment]
End to end tests: :hourglass_flowing_sand: **STARTING**

**Comment by guilhem-barthes** [comment]
/e2e --refs orchestrator=chore/network-policy --tests mnist,camelyon

**Comment by Owlfred** [comment]
End to end tests: :hourglass_flowing_sand: **STARTING**

**Comment by guilhem-barthes** [comment]
/e2e --refs orchestrator=chore/network-policy --tests sdk,substrafl,frontend,documentation,mnist,camelyon

**Comment by Owlfred** [comment]
End to end tests: :hourglass_flowing_sand: **STARTING**

**Comment by guilhem-barthes** [comment]
/e2e --refs orchestrator=chore/network-policy --tests sdk,substrafl,frontend,documentation,mnist,camelyon

**Comment by Owlfred** [comment]
End to end tests: :hourglass_flowing_sand: **STARTING**

**Comment by guilhem-barthes** [comment]
/e2e --refs orchestrator=chore/network-policy --tests substrafl,frontend,documentation

**Comment by Owlfred** [comment]
End to end tests: :hourglass_flowing_sand: **STARTING**

**Comment by Owlfred** [comment]
End to end tests: :hourglass_flowing_sand: **STARTING**

**Comment by guilhem-barthes** [comment]
/e2e --refs orchestrator=chore/network-policy --tests sdk,substrafl,frontend,documentation,mnist,camelyon

**Comment by Owlfred** [comment]
End to end tests: :hourglass_flowing_sand: **STARTING**

**Comment by guilhem-barthes** [comment]
/e2e --refs orchestrator=chore/network-policy --tests sdk,substrafl,frontend,documentation,mnist,camelyon

**Comment by Owlfred** [comment]
End to end tests: :hourglass_flowing_sand: **STARTING**

File: charts/substra-backend/templates/deployment-registry-prepopulate.yaml [hunk 1]
```diff
@@ -19,9 +19,11 @@ spec:
       labels:
         {{ include "substra.labels" $ | nindent 8 }}
         app.kubernetes.io/name: {{ template "substra.name" $ }}-registry-prepopulate-{{ $index }}
+        role-registry-client: 'true'
+        role-internet-egress: 'true'
     spec:
       initContainers:
-      - name: wait-registry
+      - name: wait-registry
```
Comment (SdgJlbl) [inline charts/substra-backend/templates/deployment-registry-prepopulate.yaml:26]: 👀

**Review by SdgJlbl — COMMENTED** [review]
LGTM

**Review by guilhem-barthes — APPROVED** [review]

**Pull request merged**